### ISSUE TITLE
Compatibility with 9.0SP2 ?

### ISSUE THREAD

**Issue by aDNers** [episerver/Ektron.Mobile.Providers#1]
Hello,

We currently use Ektron CMS 9SP2, after the intial setup we can't load smartforms in the workarea but get this message:

Error: Error loading XSLT document. http://xxx/WorkArea/ContentDesigner/ContentInComing.xslt NOTE: the SERVER is accessing this URL.

When changing back ek_EnableDeviceDetection to false, the smartforms in the workarea loads fine. 

Is the 51Degrees(TM) library verified to work with 9.0SP2? I have only read statements that it is tested with Ektron CMS 9.1. 

BR
Anders

**Comment by anielsud** [comment]
Hey Anders, this has not been tested against 9sp2, but it should work just fine. Did you update the references to the ektron dlls to the version you are building against?

**Comment by anielsud** [comment]
Hey Anders, just a quick update, I was able to reproduce your issue. I'll post a patch as soon as possible. Sorry for the inconvenience!

**Comment by aDNers** [comment]
Hello,

First of all, thanks for the quick reply!

I downloaded the dll's from the server to my computer where I have VS installed, changed the references to point to these dll's and did the build w/o errors. However, one thing differed from the instructions, in the installation guide it says "copy these 3 files to the bin-folder of the cms installation":
*Mobile.Providers.FiftyOneDegrees.dll 
*FiftyOne.Foundation.dll 
*Microsoft.Web.Infrastructure.dll 

I did only find the first two in the debug-folder of the build, the Microsoft.Web.Infrastructure.dll file was missing ( - however it already was in the bin-folder of the web server. ) Not sure if this could mean anything. 

It is when entering the "edit" mode of the content in the workarea I get the error. It doesn't matter if I try the aloha or the ewebeditpro-editor. Looking at https://developer.ektron.com/forums/?v=t&t=2679 and following the suggested KB-article does not help. (And everything seems to work ok when turning off the ek_EnableDeviceDetection.)

Any ideas are much appreciated,

BR
Anders
